bindings/ergo-lib-wasm/tests_browser/test_rest_api.js

```diff
@@ -14,13 +13,45 @@ beforeEach(async () => {
 it('node REST API: get_nipopow_proof_by_header_id endpoint', async () => {
     let node_conf = new ergo_wasm.NodeConf("213.239.193.208:9053");
     assert(node_conf != null);
-    const block_headers = generate_block_headers();
-    const header_id = block_headers.get(0).id();
+    const header_id = ergo_wasm.BlockId.from_str("4caa17e62fe66ba7bd69597afdc996ae35b1ff12e0ba90c22ff288a4de10e91b");
     let res = await ergo_wasm.get_nipopow_proof_by_header_id(node_conf, 3, 4, header_id);
     assert(res != null);
     assert(node_conf != null);
 });
 
+it('node REST API: example SPV workflow', async () => {
```

Comment:
Could you please add another node to emphasize the point that proofs should be collected from multiple nodes (per #495)?

Reply:
Done. Once #581 is resolved we can do away with hard-coded node IPs and use peer discovery to randomly pick nodes to run the workflow with.